luncheon/request.lua

```diff
@@ -1,7 +1,5 @@
 local net_url = require "net.url"
-local Headers = require "luncheon.headers"
-local utils = require "luncheon.utils"
-local shared = require "luncheon.shared"
+local ReqResp = require "luncheon.shared"
```

Comment:
I intend to rename the `shared.lua` file to `reqresp.lua`, but im really bringing it up naming to see if we want `ReqResp` to be called something else. The only other option I can think of is `Message` or `HttpMessage`, which might be better. Ill leave as is unless one other person agrees with one of the options I gave, or has a better one.

Reply:
+1 for `HttpMessage`